examples/albert_QA/README.md

```diff
@@ -0,0 +1,84 @@
+# Albert model with SQuAD dataset
+
+This project implements the Albert algorithm using the [SQuAD]() (Stanford Question Answering Dataset) for question
+answering tasks.
+
+## Population Exploration
+
+Below is a population exploration plot. It represents a samples' similarity map based on the model's latent space,
+built using the extracted features of the trained model.
+
+It shows a visualization of the training and validation sets where we can see two distinct clusters. 
+This means there is a difference in their representation—either the model does not perform well and overfits, 
```

Comment:
I suggest to rephrase this to something shorter, reducing the two options to one. (Maybe something like:  This means that there is a difference in their representation that might indicate some sort of an imbalance).